.github/workflows/summarize-impact.yaml

```diff
@@ -0,0 +1,76 @@
+name: "[TEST-IGNORE] Summarize PR Impact"
+
+on: pull_request
+
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  impact:
+    name: "[TEST-IGNORE] Summarize PR Impact"
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout eng
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            eng/
+            .github/
+
+      - name: Checkout 'after' state
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+          path: after
+
+      - name: Checkout 'before' state
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.base.sha }}
+          path: before
+
+      - name: Setup Node and install deps
+        uses: ./.github/actions/setup-node-install-deps
+
+      # todo: why is this necessary? postinstall should already take care of the build?
+      - name: Build summarize-impact tool
+        run: |
+          cd eng/tools/summarize-impact
+          npm run build
+
+      - name: Get PR labels
+        id: get-labels
+        run: |
+          labels=$(gh pr view ${{ github.event.pull_request.number }} --json labels --jq '[.labels[].name] | join(",")')
```

Comment:
I would prefer to call this API from JS instead of bash, but it might be more difficult since summarize-impact is called from the shell instead of github-script.  We could replace just this one step with github-script, but the value is questionable.  If we move all of summarize-impact to JS, we should move this as well.

Reply:
I will file an issue and follow-up on this item. For stability's sake I'm getting everything working with this set here.